packages/core/src/generators/qwik/src-generator.ts

```diff
@@ -251,7 +251,8 @@ export class SrcBuilder {
         Object.prototype.hasOwnProperty.call(bindings, rawKey) &&
         !ignoreKey(rawKey)
       ) {
-        let { code: binding } = bindings[rawKey];
+        let binding = bindings[rawKey];
+        binding = (binding && binding.code) || binding;
```

Comment:
Old habits die hard, but you can use the optional chaining operator today for this!
```suggestion
        binding = binding?.code || binding;
```

Reply:
We also got `||=`
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_OR_assignment

Reply:
fixed